src/worker/lambda/packages/packagePuller.go

```diff
@@ -36,8 +38,10 @@ type Package struct {
 	Meta         PackageMeta
 	installMutex sync.Mutex
 	installed    uint32
+	Size int
 }
 
+var totalPackageSize = 0
```

Comment:
If we have a total size variable like this, it will need to be lock protected.  Perhaps we can keep this PR simple, and figure out those details in a future PR that actually implements eviction?